### PR TITLE
Feature/team three

### PR DESCRIPTION
**Pull request by SamyJ87**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,10 +10,18 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+<<<<<<< HEAD
 
     "express": "^4.18.2",
 
     "sequelize": "^6.31.1"
 
+=======
```
Comment (deergit) [inline package.json]: this requires fixing - the second set of dependencies here is unnecessary

**Review by deergit — CHANGES_REQUESTED** [review]
please see in-code comments

File: package.json (hunk 1 above)
Comment (SamyJ87) [reply package.json]: alright

Comment (SamyJ87) [reply package.json]: update changes

**Review by deergit — APPROVED** [review]
looks good, thanks for resolving!

**Comment by deergit** [comment]
this is good for you to merge @SamyJ87

**Pull request merged**